db/sql/STANDARDS.md

```diff
@@ -62,23 +62,35 @@ Always qualify system catalog tables by prefixing them with `pg_catalog.`. If yo
 
 Always cast OID values to `bigint` before putting them in JSON (or jsonb).
 
-_Don't_ cast OID values to `integer`.
+- _Don't_ leave OID values in JSON without casting.
 
-This is because the [`oid` type](https://www.postgresql.org/docs/current/datatype-oid.html) is an _unsigned_ 32-bit integer whereas the `integer` type is a _signed_ 32-bit integer. That means it's possible for a database to have OID values which don't fit into the `integer` type.
+    This is because (strangely!) raw OID values become _strings_ in JSON unless you cast them.
```

Comment:
This is something I've noticed then forgotten at least 5 times. Probably a good idea to have it written down.